docs/IAMPERMISSIONS.md

```diff
@@ -0,0 +1,249 @@
+# Limiting IAM Permissions
+
+It is possible to define a role limiting IAM permissions when deploying to Amazon EKS or ECS.
+It is a best practice to use roles providing the minimum required permissions.
+The settings below have been found to be a good basis for assigning these minimum permissions when running `testsys` tests.
+
+## EKS
+
+The following is a minimal set of permissions when deploying to an EKS cluster:
```

Comment:
This seems like a lot just to get TestSys running in an EKS cluster.

If I understand this correctly, this seems like the IAM permissions necessary to _create_ an EKS cluster. In my mind, that seems to be out of the TestSys scope, because TestSys can be deployed to any K8s cluster and it definitely doesn't need to be an EKS cluster.

I think what would be interesting is to provide set of minimum IAM permissions for the worker nodes' instance profile or things a user would have to do to run some of the common tests TestSys offers; Specifically, something like steps to allow test agents to be able to dispatch K8s conformance tests targeting external EKS clusters.

Reply:
Maybe reword this to something like "when creating and deploying to an EKS cluster"?

Today if someone follows our quickstart docs, they do need the permissions to be able to create the cluster. It's possible to use an existing one, but for someone new coming in and following those docs, they do need these permissions.

Maybe break out into a separate section that calls this out?